src/App.js

```diff
@@ -28,7 +29,8 @@ function App() {
     <div className="App">
       {location.pathname === "/home" ||
       location.pathname === "/archived" ||
-      location.pathname === "/label" ? (
+      location.pathname === "/label" ||
+      location.pathname === "/trash" ? (
         <Header />
```

Comment:
Instead of using is conditional operator you can use ternary operator as there is nothing to return if the condition is false. also if the path of where you want Header are more than where you dont want Header you can use !== instead of === for those paths, it will reduce your code in some amount.

Reply:
Yep will change this according to ankith's suggestion